build.py

```diff
@@ -838,6 +840,30 @@ def install_dcgm_libraries(dcgm_version, target_machine):
 '''.format(dcgm_version, dcgm_version)
 
 
+def install_miniconda(conda_version, target_machine):
+    if conda_version == '':
+        fail(
+            'unable to determine default repo-tag, CONDA version not known for {}'
+            .format(FLAGS.version))
+    miniconda_url = f"https://repo.anaconda.com/miniconda/Miniconda3-{conda_version}-Linux-{target_machine}.sh"
+    if target_machine == 'x86_64':
+        sha_sum = "3190da6626f86eee8abf1b2fd7a5af492994eb2667357ee4243975cdbb175d7a"
+    else:
+        sha_sum = "47affd9577889f80197aadbdf1198b04a41528421aaf0ec1f28b04a50b9f3ab8"
```

Comment:
Just for my own curiosity, will these SHA sums have to be updated whenever the conda_version is bumped?

Reply:
> have to be updated whenever the conda_version is bumped?
Yes. I added a comment.